### DISCUSSION TITLE
控制台的移动端识别对windows平板可能存在误判

### DISCUSSION THREAD

**Discussion by koldle** [zeabur/.github#21] [General]
设备: 小米平板2
OS: Win11

设备是x86架构的, 打开zeabur控制台被识别成移动端😫感觉怪怪的, 浏览其他网页没被识别成移动端过

**Comment by koldle** [comment]
看起来是根据屏幕比例判断? 同样是rdp到远程电脑, 手机横屏就能正常进入, 小米平板2这4:3的屏幕就只能被识别成移动端🤣

**Reply by MichaelYuhe** [reply]
是根据屏幕宽度判断的噢，后续会开发移动端的控制台
